src/LiveComponent/doc/index.rst

```diff
@@ -2523,41 +2523,136 @@ To change the initial tag from a ``div`` to something else, use the ``loading-ta
+                console.log(`Polling started: ${actionName}, limit: ${limit}`);
+            });
+
+            this.component.on('poll:running', ({ actionName, count, limit }) => {
+                console.log(`Polling running: ${actionName} (${count}/${limit})`);
+            });
+
+            this.component.on('poll:paused', ({ actionName, count, limit }) => {
+                console.log(`Polling paused: ${actionName}`);
+            });
+
+            this.component.on('poll:stopped', ({ actionName, finalCount, limit }) => {
+                console.log(`Polling stopped: ${actionName}, total runs: ${finalCount}`);
+            });
+
+            this.component.on('poll:error', ({ actionName, finalCount, limit, errorMessage }) => {
+                console.error(`Polling error on ${actionName}: ${errorMessage}`);
+            });
+        }
+    }
+
+.. note::
+
+    These events are dispatched on the component and can be handled using Stimulus. You must retrieve the component instance with ``getComponent(this.element)`` before accessing event listeners.
+
+Handling Poll Actions (Start, Stop, Pause, Resume)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Starting from version 2.29, polling can be programmatically managed using the ``pollingDirector`` API.
+
+This allows you to start, pause, resume, or stop polling dynamically for a given action.
 
 .. code-block:: html+twig
 
-    <div
-        {{ attributes }}
+    <div {{ attributes.defaults(stimulus_controller('poll')) }} data-poll="delay(5000)|limit(50)|$render">
+        <button type="button" data-action="click->poll#start" data-poll-action-param="$render">Start</button>
+        <button type="button" data-action="click->poll#stop" data-poll-action-param="$render">Stop</button>
+    </div>
 
-        data-poll="save"
-        {#
-        Or add a delay() modifier:
-        data-poll="delay(2000)|save"
-        #}
-    >
+.. code-block:: javascript
+
+    // controllers/poll_controller.js
+    import { Controller } from '@hotwired/stimulus';
+    import { getComponent } from '@symfony/ux-live-component';
+
+    export default class extends Controller {
+        static values = {
+            action: String
+        }
+
+        async start(event) {
+            const actionName = event.params.action;
+            (await getComponent(this.element)).pollingDirector.start(actionName);
+        }
+
+        async stop(event) {
+            const actionName = event.params.action;
+            (await getComponent(this.element)).pollingDirector.stop(actionName);
+        }
+
+        async pause(event) {
+            const actionName = event.params.action;
+            (await getComponent(this.element)).pollingDirector.pause(actionName);
+        }
+
+        async resume(event) {
+            const actionName = event.params.action;
+            (await getComponent(this.element)).pollingDirector.resume(actionName);
+        }
+    }
```

Comment:
I'm not sure we need as many full examples in the documentation. 

Could we just list the methods/hooks for both, and use one example to illustrate how to use a Stimulus controller to listen to these events/actions ?